src/components/tooltip/tooltip.test.js

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
+import * as React from "react";
+import { forwardRef } from "react";
+import Tooltip from "./tooltip.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import { tooltipPreview } from "../../../cypress/locators/tooltip/index";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const backgroundColors = [
```

Comment:
It might be an idea in future to add hex values and design-token values in to this array to assert against. Just depends if @DlgSHi thinks it is worth it.

Reply:
It is not possible to check it using hex/desigtokens :(

Reply:
Not to worry then. I'll approve the PR.